packages/cli/src/lib/commands/space.ts

```diff
@@ -148,17 +148,33 @@ export const space: CommandDefinition = (program) => {
 
```

Comment:
Please get rid of `Analyze the source code` errors while your here ;)  I know they're not yours but annoys ppl :)